src/test/java/basesyntax/strategy/BalanceOperationHandlerImplTest.java

```diff
@@ -0,0 +1,36 @@
+package basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import basesyntax.dao.StorageDao;
+import basesyntax.dao.StorageDaoImpl;
+import basesyntax.db.Storage;
+import basesyntax.model.FruitTransaction;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerImplTest {
+    private static OperationHandler operationHandler;
+
+    @BeforeClass
+    public static void beforeClass() {
+        StorageDao storageDao = new StorageDaoImpl();
+        operationHandler = new BalanceOperationHandlerImpl(storageDao);
+    }
+
+    @Test
+    public void balance_operation_Ok() {
```

Comment:
```suggestion
    public void changeQuantity_balanceOperation_ok() {
```
Let's fix naming everywhere